clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fmap.py

```diff
@@ -1,11 +1,17 @@
 """Module for converting field maps of ADNI."""
+import json
+import os
+import re
+import shutil
+from enum import StrEnum, auto
```

Comment:
I didn't realize it when reviewing, but `StrEnum` is only available in Python 3.11 onward (see https://docs.python.org/3/library/enum.html#enum.StrEnum).
Which is explaining all these unit test failures.

Since we support Python 3.9 onward, I would vote for changing the implementation:

```python
from enum import Enum

class BIDSFMAPCase(str, Enum):
    CASE1 = "case1"
    CASE2 = "case2"
    ...
```

Reply:
Ok, no problem, I will change that !

The PR should not be merged yet though, I noticed the conversion fails when trying to convert all modalities (but succeeds with FMAP only)